pkg/storage/unified/sql/sqltemplate/dialect_sqlite.go

```diff
@@ -16,3 +16,12 @@ type sqlite struct {
 	argPlaceholderFunc
 	name
 }
+
+func (sqlite) CurrentEpoch() string {
+	// Alternative approaches like `unixepoch('subsecond') * 1000000` returns millisecond precision.
+	return "CAST((julianday('now') - 2440587.5) * 86400000000.0 AS INTEGER)"
```

Comment:
```suggestion
	return "CAST((julianday('now') - 2440587.5) * 86400000000.0 AS BIGINT)"
```
This suggestion doesn't really affect the outcome since SQLite is not strongly typed, so even using `SMALLINT` works lol (try it! `echo "select CAST((julianday('now') - 2440587.5) * 86400000000.0 AS SMALLINT);" | sqlite3`), but it's good as a way of documenting the intention at least. As a reference, the maximum value that can be represented with a 4 bytes signed integer is 2147483647, which is far less than something like 1727281095078013, which is the current timestamp with microseconds as of this writing.